src/components/tutorials/VideoPlayer.js

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import Video from "react-native-video";
+import {ScrollView, Text, View, Image, TouchableWithoutFeedback} from "react-native";
+import {Icon, Spinner} from 'native-base';
+import {VIDEO_PLAYER_STYLES} from "../../styles/tutorials/styles";
+import {spinnerColor} from "../../styles/main/styles";
+
+export class VideoPlayer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            loaded: false,
+            paused: false,
+            showingInFullScreen: false,
+            videoStyle: VIDEO_PLAYER_STYLES.videoLoading,
+            videoContainerStyle: VIDEO_PLAYER_STYLES.videoContainer,
+            controlsContainerStyle: VIDEO_PLAYER_STYLES.controlsContainer
+        };
+
+        this._renderSpinner = this._renderSpinner.bind(this);
+        this.onVideoLoaded = this.onVideoLoaded.bind(this);
+        this._renderPlayerControls = this._renderPlayerControls.bind(this);
+        this.pause = this.pause.bind(this);
+        this.play = this.play.bind(this);
+        this.expandScreen = this.expandScreen.bind(this);
+        this.shrinkScreen = this.shrinkScreen.bind(this);
+        this._renderOverlay = this._renderOverlay.bind(this);
```

Comment:
@MaxiSuppes Los métodos de "_render" no necesitan ser bindeados porque se los llama normalmente. Sólo se necesitan bindear los métodos que se pasan como objeto (estilo this.metodo) en alguna parte (los que normalmente empiezan con "handle").

Reply:
@lucianolev Cuando tengo muchos metodos hago el bind de todos como una especie de documentación. Para ver fácilmente todos los métodos de ese componente